app/src/main/java/pm/gnosis/heimdall/ui/safe/recover/safe/RecoverSafe2FAActivity.kt

```diff
@@ -2,15 +2,26 @@ package pm.gnosis.heimdall.ui.safe.recover.safe
 
 import android.content.Context
 import android.content.Intent
-import kotlinx.coroutines.ExperimentalCoroutinesApi
+import android.os.Bundle
+import kotlinx.android.synthetic.main.layout_select_authenticator.*
 import pm.gnosis.heimdall.ui.two_factor.Select2FaActivity
+import pm.gnosis.heimdall.utils.AuthenticatorInfo
 import pm.gnosis.heimdall.utils.AuthenticatorSetupInfo
 import pm.gnosis.model.Solidity
+import pm.gnosis.svalinn.common.utils.visible
 import pm.gnosis.utils.asEthereumAddress
 import pm.gnosis.utils.asEthereumAddressString
 
 class RecoverSafe2FAActivity : Select2FaActivity() {
 
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        //FIXME: allow to select Gnosis Authenticator when recovering for now
```

Comment:
Why is that a `FIXME` ... makes it sound like a bug :P

Reply:
something that has to be changed in the future :) I'll change it to normal comment